docs/components/props.md

```diff
@@ -1,30 +1,30 @@
 ```python exec
-from pcweb.pages.docs.library import library
-from pcweb.pages.docs import state, vars
 import reflex as rx
+from pcweb.pages.docs.library import library
+from pcweb.pages import docs
 ```
 
 # Props
 
-Props modify the behavior and appearance of a component. They are passed in as keyword arguments to the component function.
+Props modify the behavior and appearance of a component. They are passed in as keyword arguments to a component.
 
 ## Component Props
 
-Each component has props that are specific to that component. For example, the `rx.avatar` component has a fallback prop that sets the `fallback` of the avatar.
+There are props that are shared between all components, but each component can also define its own props.
+
+For example, the `rx.image` component has a `src` prop that specifies the URL of the image to display and an `alt` prop that specifies the alternate text for the image.
 
 ```python demo
-rx.avatar(
-    fallback="JD"
+rx.image(
+    src="https://reflex.dev/logo.jpg",
+    alt="Reflex Logo",
 )
 ```
 
-Check the docs for the component you are using to see what props are available.
+Check the docs for the component you are using to see what props are available and how they affect the component (see the `rx.image` [reference]({docs.library.media.image.path}#api-reference) page for example).
 
-```md alert success
-# Reflex has a wide selection of [built-in components]({library.path}) to get you started quickly.
-```
 
-## HTML Props
+## Common Props
 
 Components support many standard HTML properties as props. For example: the HTML [id]({"https://www.w3schools.com/html/html_id.asp"}) property is exposed directly as the prop `id`. The HTML [className]({"https://www.w3schools.com/jsref/prop_html_classname.asp"}) property is exposed as the prop `class_name` (note the Pythonic snake_casing!).
 
```

Comment:
might be nice for this example to actually define class-name-1 and class-name-2 so it is a bit more tangible what is happening here, as people coming from a non-web dev background might be like what is happening in this example?

Reply:
don't see any change here?

Reply:
When you say define, what do you mean? I put out an explanation on line 39 explaining what class names will do:

> In the example above, the `class_name` prop of the `rx.box` component is assigned a list of class names. This means the `rx.box` component will be styled with the CSS classes `class-name-1` and `class-name-2`.

Is this not what we want?

Reply:
in this case I meant we could actually add the class-name-1 with the values that it takes, but it is a small point so let's leave it for now